### DISCUSSION TITLE
如何导出下载的漫画

### DISCUSSION THREAD

**Discussion by Z-fly** [xiaoyaocz/flutter_dmzj#164] [General]
如题

**Comment by donmor** [comment]
目前据我试验linux版/安卓版下载位置分别在`~/.local/share/com.xvcz.dmzjx`和`/data/data/com.xycz.dmzjx/files`下（后者需要备份或root），以`<作品号>_<章节号>/xxx.jpg`方式保存，要想导出估计需要用dmzj的api请求到作品/章节名然后压缩cbz

**Reply by Z-fly** [reply]
@xiaoyaocz
